uint8 id_task
uint8 number_ports
port[] ports
